apps/web/src/app/admin/cohorts/[id]/page-client.tsx

```
@@ -954,125 +973,126 @@ export function CohortDetailPageClient({
-													<span className="font-mono text-sm">
-														{formatTime(session.start_time)} -{" "}
-														{formatTime(session.end_time)}
-													</span>
-												</div>
+								{cohortWithSessions?.weekly_sessions?.map((session: any) => (
+									<div
+										key={session.id}
+										className="group relative cursor-pointer overflow-hidden rounded-lg border bg-card transition-all duration-200 hover:shadow-md"
+										onClick={() => handleEditSession(session)}
+										role="button"
+										tabIndex={0}
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)